components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2295,17 +2299,39 @@ impl Apply {
         commit_term: u64,
         commit_index: u64,
     ) -> Apply {
+        let entries_mem_size = get_entries_mem_size(&entries);
+        APPLY_PENDING_BYTES_GAUGE
+            .add((ENTRY_MEM_SIZE * entries.capacity()) as i64 + entries_mem_size);
```

Comment:
Can the capacity be also added to `entries_mem_size`?

Reply:
Fixed.